app/views/articles/_video_player.html.erb

```diff
@@ -1,18 +1,26 @@
-<div itemscope itemtype="http://schema.org/VideoObject" class="video-player-header">
+<div class="video-player-header">
   <% if meta_tags %>
-    <meta itemprop="uploadDate" content="<%= article.published_at %>" />
-    <meta itemprop="name" content="<%= article.title %>" />
-    <meta itemprop="description" content="<%= article.description %>" />
-    <meta itemprop="thumbnailUrl" content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
-    <meta itemprop="contentUrl" content="<%= article.video_source_url %>" />
     <% minutes, seconds = article.video_duration_in_minutes.split(":") %>
-    <meta itemprop="duration" content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
     <div id="video-player-source" data-source="<%= article.video_source_url %>"></div>
   <% end %>
   <script src="//content.jwplatform.com/libraries/b1zWy2iv.js" async></script>
   <div id="video-player-<%= article.id %>" class="video-player"></div>
 </div>
 
+<script type="application/ld+json">
+  {
+    "@context": "https://schema.org",
+    "@type": "VideoObject",
+    "url": "<%= article.video_source_url %>",
+    "uploadDate": "<%= article.published_at %>",
```

Comment:
Maybe we can change this to `article.published_at&.rfc3339` to render a RFC compliant date string, what do you think?

BTW is the name arbitrary? If so maybe we can change it to `datePublished` like we do in `article/show.html.rb`, don't know 🤷‍♂